index.html

```diff
@@ -0,0 +1,295 @@
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+    crossorigin="anonymous">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <link rel="icon" href="favicon.ico" type="image/x-icon">
+  <title>Free Videos</title>
+
+</head>
+
+<body>
+  <section class="full-container banner">
+    <video autoplay muted loop class="myVideo">
+      <source src="./assets/videos/video-loop-1.mp4" type="video/mp4">
+    </video>
+    <div class="navbar">
+      <nav class="navbar-nav">
+        <ul>
+          <li class="nav-items">
+            <a href="#">License</a>
+          </li>
+          <li class="nav-items">
+            <a href="#">Popular Videos</a>
+          </li>
+          <li class="nav-items">
+            <a href="#">Free Photos</a>
+          </li>
+          <li class="nav-items">
+            <a href="#">
+              <i class="fas fa-ellipsis-h"></i>
+            </a>
+          </li>
+          <li class="nav-items menu-clicker">
+            <a href="#">
+              <i class="fas fa-bars"></i>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <article class="header-center">
+      <img src="./assets/images/pexels-logo.png" alt="pexels">
+      <p> Completely free stock videos.
+        <a href="#" class="learn-more">Learn More</a>
+      </p>
+      <form action="submit" method="POST">
+        <input type="text" placeholder="Search for free video">
+        <button type="button">
+          <i class="fas fa-search"></i>
+        </button>
+      </form>
+      <h3>Popular Searches:</h3>
+      <p class="popular-search-links">
+        <a href="#">fire</a>
+        <a href="#">city</a>
+        <a href="#">business</a>
+        <a href="#">street</a>
+        <a href="#">running</a>
+        <a href="#">writing</a>
+        <a href="#">iphone</a>
+        <a href="#">tech</a>
+        <a href="#">adventure</a>
+        <a href="#">landscape</a>
+        <a href="#">cooking</a>
+        <a href="#">cat</a>
+        <a href="#">woman</a>
+        <a href="#">water</a>
+      </p>
+    </article>
+    <div class="container">
+      <article class="header-bottom">
+        <div class="left-content">
+          <i class="fas fa-thumbs-up">Like</i>
+
+          <i class="fab fa-twitter">Follow</i>
+        </div>
+        <div class="right-content">
+          <a href="#">Free Stock Photos
+            <i class="fas fa-long-arrow-alt-right"></i>
+          </a>
+        </div>
+      </article>
+    </div>
+
+  </section>
+  <br class="clear">
+  <section class="container-sm">
+    <h2>Popular Videos</h2>
+    <div class="three-columns">
```

Comment:
Just a heads up, I'm getting different behavior depending on the browser and the browser window width (Firefox displays 4 columns)